dcos/http.py

```diff
@@ -13,7 +13,18 @@
 
 logger = util.get_logger(__name__)
 
-DEFAULT_TIMEOUT = 5
+DEFAULT_READ_TIMEOUT = 180
```

Comment:
It is probably useful to link to http://docs.python-requests.org/en/master/user/advanced/#timeouts somewhere.